<ui version="4.0" >
 <class>PlaylistPreferences</class>
 <widget class="QDialog" name="PlaylistPreferences" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Playlist - Preferences</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QCheckBox" name="play_from_start_check" >
     <property name="text" >
      <string>&amp;Play files from start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="recursive_check" >
     <property name="text" >
      <string>&amp;Add files in directories recursively</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="getinfo_check" >
     <property name="text" >
      <string>Automatically get &amp;info about files added</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autosave_on_exit_check" >
     <property name="text" >
      <string>&amp;Save copy of playlist on exit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlaylistPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlaylistPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
